callisto/notification/api.py

```diff
@@ -0,0 +1,95 @@
+import logging
+
+import gnupg
+import pytz
+
+from django.conf import settings
+from django.core.mail.message import EmailMultiAlternatives
+from django.utils import timezone
+
+from callisto.delivery.models import SentMatchReport
+from callisto.notification.models import EmailNotification
+
+logger = logging.getLogger(__name__)
+tzname = settings.REPORT_TIME_ZONE or 'America/Los_Angeles'
+timezone.activate(pytz.timezone(tzname))
+
+
+class NotificationApi(object):
```

Comment:
I gotcha, I imagine the override could be something like `settings.CALLISTO_NOTIFICATION_API=...`

And I agree with the notification / delivery split, although I'm not 100% on how I would implement something like that